README.md

```diff
@@ -1,7 +1,60 @@
 # Advanced Solutions Lab
 
-This repository contains labs and solutions for the Advanced Solutions Lab: ML Immersion.
+## Overview
+This repository contains Jupyter notebooks meant to be run on Vertex AI. This is maintained by Google Cloud’s [Advanced Solutions Lab (ASL)](https://cloud.google.com/asl) team. [Vertex AI](https://cloud.google.com/vertex-ai) is the next generation AI Platform on the Google Cloud Platform.
+The material covered in this repo will take a software engineer with no exposure to machine learning to an advanced level.  
+
+In particular, the notebooks in this repository cover
+- A wide range of model architectures (DNN, CNN, RNN, transformers, SNGP, etc.) targeting many data modalities (tabular, image, text, time-series) implemented mainly in Tensorflow and Keras.
+- Tools on Google Cloud’s Vertex AI for operationalizing Tensorflow, Scikit-learn and PyTorch models at scale.
```

Comment:
at scale. --> at scale (e.g. Vertex training, tuning, and serving, TFX and Kubeflow pipelines).